internal/creds.go

```diff
@@ -47,6 +47,8 @@ func Creds(ctx context.Context, ds *DialSettings) (*google.Credentials, error) {
 // options provided via [option.ClientOption], including legacy oauth2/google
 // options. If there are no applicable options, then it returns the result of
 // [cloud.google.com/go/auth/credentials.DetectDefault].
+// Note: If NoAuth is true(when option.WithoutAuthentication is passed), then no
```

Comment:
Nit: should this be linked like others in this commentary?
```suggestion
// Note: If NoAuth is true(when [option.WithoutAuthentication] is passed), then no
```

Reply:
done.